### PR TITLE
Changelog for 1.2.2

### PR DESCRIPTION
**Pull request by minrk**
backport a few bugfixes, mostly for regressions in 1.2.0

### CODE REVIEW

File: docs/source/changelog.md [hunk 1]
```diff
@@ -9,6 +9,40 @@ command line for details.
...
+* remove push-branch conditions for CI [#3250](https://github.com/jupyterhub/jupyterhub/pull/3250) ([@minrk](https://github.com/minrk))
+* Migrate from travis to GitHub actions [#3246](https://github.com/jupyterhub/jupyterhub/pull/3246) ([@consideRatio](https://github.com/consideRatio))
+
+#### Documentation improvements
+
+* Update services-basics.md to use jupyterhub_idle_culler [#3257](https://github.com/jupyterhub/jupyterhub/pull/3257) ([@manics](https://github.com/manics))
+
+#### Other merged PRs
+
+* Environment marker on pamela [#3255](https://github.com/jupyterhub/jupyterhub/pull/3255) ([@fcollonval](https://github.com/fcollonval))
```
Comment (consideRatio) [inline docs/source/changelog.md:37]: Moved the single "Other merged PRs" about pamela to Maintenance/upkeep and labelled that PR maintenance.

```suggestion
#### Maintenance and upkeep improvements

* Environment marker on pamela [#3255](https://github.com/jupyterhub/jupyterhub/pull/3255) ([@fcollonval](https://github.com/fcollonval))
* remove push-branch conditions for CI [#3250](https://github.com/jupyterhub/jupyterhub/pull/3250) ([@minrk](https://github.com/minrk))
* Migrate from travis to GitHub actions [#3246](https://github.com/jupyterhub/jupyterhub/pull/3246) ([@consideRatio](https://github.com/consideRatio))

#### Documentation improvements

* Update services-basics.md to use jupyterhub_idle_culler [#3257](https://github.com/jupyterhub/jupyterhub/pull/3257) ([@manics](https://github.com/manics))
```

**Comment by consideRatio** [comment]
This seems sensible to me! +1 for merge and release

**Comment by minrk** [comment]
@MeeSeeksDev backport to 1.2.x

**Comment by minrk** [comment]
published 1.2.2!

**Pull request merged**